tests/Functional/Cases/Parsing/NamespaceRegistrationTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace TYPO3Fluid\Fluid\Tests\Functional\Cases\Parsing;
+
+use TYPO3Fluid\Fluid\Tests\Functional\BaseFunctionalTestCase;
+
+/**
+ * Class NamespaceRegistrationTest
+ */
+class NamespaceRegistrationTest extends BaseFunctionalTestCase
+{
+
+    /**
+     * @var array
+     */
+    protected $variables = [];
+
+    /**
+     * @return array
+     */
+    public function getTemplateCodeFixturesAndExpectations()
+    {
+        return [
+            'Ignoring namespaces without conflict with registered namespace' => [
+                '{namespace z*}{namespace bar}<zoo:bar /><bar:foo></bar:foo><zoo.bar:baz /><f:format.raw>foobar</f:format.raw>',
+                $this->variables,
+                ['<zoo.bar:baz />'],
```

Comment:
Reading these test cases:

* First array is an array of (string) values that *must appear in output when executed*.
* Second array is an array of (string) values that *must NOT appear in output when executed*.